reference/docs-conceptual/samples/Working-with-Printers.md

```diff
@@ -41,7 +41,8 @@ To use WMI to set the default printer, find the printer in the **Win32_Printer**
 invoke the **SetDefaultPrinter** method:
 
 ```powershell
-(Get-CimInstance -Class Win32_Printer -Filter "Name='HP LaserJet 5Si'").SetDefaultPrinter()
+$printer = Get-CimInstance -Class Win32_Printer -Filter "Name='HP LaserJet 5Si'")
```

Comment:
Looks like an erroneous trailing character on this line.

Reply:
Fixed